tests/integration_test.rs

```diff
@@ -8,8 +8,8 @@ use std::{
 
```

Comment:
Could you add an integration test?

Reply:
Added with [38864f5](https://github.com/asimsedhain/solace-rs/pull/27/commits/38864f5ae9528dc762e431f214b4e07318d71c3d)